From af28b827a9577674fd1aef2110338bdc1d1783c0 Mon Sep 17 00:00:00 2001
From: Andrew Lunn <andrew@lunn.ch>
Date: Mon, 5 May 2025 18:39:12 -0500
Subject: [PATCH 15/30] net: phy: maxio: Consistently check return codes

Signed-off-by: Andrew Lunn <andrew@lunn.ch>
---
 drivers/net/phy/maxio.c | 20 +++++++++++++-------
 1 file changed, 13 insertions(+), 7 deletions(-)

diff --git a/drivers/net/phy/maxio.c b/drivers/net/phy/maxio.c
index 364eb6b13..361b19b64 100755
--- a/drivers/net/phy/maxio.c
+++ b/drivers/net/phy/maxio.c
@@ -47,26 +47,30 @@ static int maxio_mae0621a_clk_init(struct phy_device *phydev)
 	//get workmode
 	workmode = phy_read_paged(phydev, 0xa43,
 				  MAXIO_MAE0621A_WORK_STATUS_REG);
+	if (workmode < 0)
+		return workmode;
 
 	//get clkmode
 	clkmode = phy_read_paged(phydev, 0xd92, MAXIO_MAE0621A_CLK_MODE_REG);
+	if (clkmode < 0)
+		return clkmode;
 
 	//abnormal
 	if (0 == (workmode&BIT(5))) {
 		if (0 == (clkmode&BIT(8))) {
 			//oscillator
-			phy_write_paged(phydev, 0xd92,
-					MAXIO_MAE0621A_CLK_MODE_REG,
-					clkmode | BIT(8));
+			ret = phy_write_paged(phydev, 0xd92,
+					      MAXIO_MAE0621A_CLK_MODE_REG,
+					      clkmode | BIT(8));
 		} else {
 			//crystal
-			phy_write_paged(phydev, 0xd92,
-					MAXIO_MAE0621A_CLK_MODE_REG,
-					clkmode &(~ BIT(8)));
+			ret = phy_write_paged(phydev, 0xd92,
+					      MAXIO_MAE0621A_CLK_MODE_REG,
+					      clkmode &(~ BIT(8)));
 		}
 	}
 
-	return 0;
+	return ret;
 }
 
 static int maxio_mae0621a_config_init(struct phy_device *phydev)
@@ -81,6 +85,8 @@ static int maxio_mae0621a_config_init(struct phy_device *phydev)
 
 	//enable auto_speed_down
 	ret = phy_write_paged(phydev, 0xd8f, 0x0, 0x300 );
+	if (ret < 0)
+		return ret;
 
 	//adjust TX/RX delay
 	switch (phydev->interface) {
-- 
2.39.5

