From bf1023ec4009e388beffc2fc217bf4ec3615cdba Mon Sep 17 00:00:00 2001
From: Andrew Lunn <andrew@lunn.ch>
Date: Mon, 5 May 2025 18:47:29 -0500
Subject: [PATCH 19/30] net: phy: maxio: Whitespace cleanup

Signed-off-by: Andrew Lunn <andrew@lunn.ch>
---
 drivers/net/phy/maxio.c | 47 ++++++++++++++++++++++-------------------
 1 file changed, 25 insertions(+), 22 deletions(-)

diff --git a/drivers/net/phy/maxio.c b/drivers/net/phy/maxio.c
index 21e2c3609..31789b756 100755
--- a/drivers/net/phy/maxio.c
+++ b/drivers/net/phy/maxio.c
@@ -16,14 +16,14 @@
 #include <linux/timer.h>
 #include <linux/netdevice.h>
 
-#define MAXIO_PAGE_SELECT			    0x1f
-#define MAXIO_MAE0621A_INER				0x12
+#define MAXIO_PAGE_SELECT		0x1f
+#define MAXIO_MAE0621A_INER		0x12
 #define MAXIO_MAE0621A_INER_LINK_STATUS	BIT(4)
-#define MAXIO_MAE0621A_INSR				0x1d
-#define MAXIO_MAE0621A_TX_DELAY			(BIT(6)|BIT(7))
-#define MAXIO_MAE0621A_RX_DELAY			(BIT(4)|BIT(5))
-#define MAXIO_MAE0621A_CLK_MODE_REG      0x02
-#define MAXIO_MAE0621A_WORK_STATUS_REG   0x1d
+#define MAXIO_MAE0621A_INSR		0x1d
+#define MAXIO_MAE0621A_TX_DELAY		(BIT(6) | BIT(7))
+#define MAXIO_MAE0621A_RX_DELAY		(BIT(4) | BIT(5))
+#define MAXIO_MAE0621A_CLK_MODE_REG     0x02
+#define MAXIO_MAE0621A_WORK_STATUS_REG  0x1d
 
 static int maxio_read_page(struct phy_device *phydev)
 {
@@ -37,7 +37,7 @@ static int maxio_write_page(struct phy_device *phydev, int page)
 
 static int maxio_mae0621a_clk_init(struct phy_device *phydev)
 {
-	u32 workmode,clkmode;
+	u32 workmode, clkmode;
 	int ret;
 
 	ret = genphy_soft_reset(phydev);
@@ -53,8 +53,8 @@ static int maxio_mae0621a_clk_init(struct phy_device *phydev)
 	if (clkmode < 0)
 		return clkmode;
 
-	if (0 == (workmode&BIT(5))) {
-		if (0 == (clkmode&BIT(8))) {
+	if (0 == (workmode & BIT(5))) {
+		if (0 == (clkmode & BIT(8))) {
 			//oscillator
 			ret = phy_write_paged(phydev, 0xd92,
 					      MAXIO_MAE0621A_CLK_MODE_REG,
@@ -63,7 +63,7 @@ static int maxio_mae0621a_clk_init(struct phy_device *phydev)
 			//crystal
 			ret = phy_write_paged(phydev, 0xd92,
 					      MAXIO_MAE0621A_CLK_MODE_REG,
-					      clkmode &(~ BIT(8)));
+					      clkmode & ~BIT(8));
 		}
 	}
 
@@ -81,7 +81,7 @@ static int maxio_mae0621a_config_init(struct phy_device *phydev)
 	phy_disable_eee(phydev);
 
 	/* enable downshift */
-	ret = phy_write_paged(phydev, 0xd8f, 0x0, 0x300 );
+	ret = phy_write_paged(phydev, 0xd8f, 0x0, 0x300);
 	if (ret < 0)
 		return ret;
 
@@ -111,13 +111,13 @@ static int maxio_mae0621a_config_init(struct phy_device *phydev)
 
 	ret &= ~(MAXIO_MAE0621A_TX_DELAY | MAXIO_MAE0621A_RX_DELAY);
 
-	ret = phy_write_paged(phydev, 0xd96, 0x0, val|ret );
+	ret = phy_write_paged(phydev, 0xd96, 0x0, val | ret);
 	if (ret < 0) {
 		dev_err(dev, "Failed to update the TX delay register\n");
 		return ret;
 	}
-delay_skip:
 
+delay_skip:
 	ret = genphy_soft_reset(phydev);
 	if (ret < 0)
 		return ret;
@@ -125,10 +125,11 @@ static int maxio_mae0621a_config_init(struct phy_device *phydev)
 	return 0;
 }
 
-
 static int maxio_mae0621a_resume(struct phy_device *phydev)
 {
-	int ret = genphy_resume(phydev);
+	int ret;
+
+	ret = genphy_resume(phydev);
 	if (ret < 0)
 		return ret;
 
@@ -137,23 +138,25 @@ static int maxio_mae0621a_resume(struct phy_device *phydev)
 
 static int maxio_mae0621a_probe(struct phy_device *phydev)
 {
-	int ret = maxio_mae0621a_clk_init(phydev);
+	int ret;
+
+	ret = maxio_mae0621a_clk_init(phydev);
 	mdelay(100);
 	return ret;
 }
 
 static struct phy_driver maxio_nc_drvs[] = {
 	{
-		.phy_id			= 0x7b744411,
+		.phy_id		= 0x7b744411,
 		.phy_id_mask	= 0x7fffffff,
-		.name			= "MAE0621A Gigabit Ethernet",
-		.features		= PHY_GBIT_FEATURES,
-		.probe			= maxio_mae0621a_probe,
+		.name		= "MAE0621A Gigabit Ethernet",
+		.features	= PHY_GBIT_FEATURES,
+		.probe		= maxio_mae0621a_probe,
 		.config_init	= maxio_mae0621a_config_init,
 		.config_aneg	= genphy_config_aneg,
 		.read_status	= genphy_read_status,
 		.suspend	= genphy_suspend,
-		.resume			= maxio_mae0621a_resume,
+		.resume		= maxio_mae0621a_resume,
 		.read_page	= maxio_read_page,
 		.write_page	= maxio_write_page,
 	},
-- 
2.39.5

