From 88334290145aa8b78fba4e4bd96f3e154e481217 Mon Sep 17 00:00:00 2001
From: Andrew Lunn <andrew@lunn.ch>
Date: Mon, 5 May 2025 18:47:29 -0500
Subject: [PATCH 21/29] net: phy: maxio: Add missing device table to module
 loading works

Signed-off-by: Andrew Lunn <andrew@lunn.ch>
---
 drivers/net/phy/maxio.c | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/drivers/net/phy/maxio.c b/drivers/net/phy/maxio.c
index c8ef72f7a..6d4cd4984 100755
--- a/drivers/net/phy/maxio.c
+++ b/drivers/net/phy/maxio.c
@@ -162,6 +162,13 @@ static struct phy_driver maxio_nc_drvs[] = {
 };
 module_phy_driver(maxio_nc_drvs);
 
+static const struct mdio_device_id __maybe_unused maxio_tbl[] = {
+	{ PHY_ID_MATCH_EXACT(0x7b744411) },
+	{ }
+};
+
+MODULE_DEVICE_TABLE(mdio, maxio_tbl);
+
 MODULE_DESCRIPTION("Maxio PHY driver");
 MODULE_AUTHOR("Zhao Yang");
 MODULE_LICENSE("GPL");
-- 
2.39.5

