From b6a1452bb235ac27f5e9219fc3bd2ba37131a825 Mon Sep 17 00:00:00 2001
From: Andrew Lunn <andrew@lunn.ch>
Date: Mon, 5 May 2025 18:47:29 -0500
Subject: [PATCH 22/30] net: phy: maxio: Use C comment style, not C++

Signed-off-by: Andrew Lunn <andrew@lunn.ch>
---
 drivers/net/phy/maxio.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/net/phy/maxio.c b/drivers/net/phy/maxio.c
index 6d4cd4984..6d2885a5f 100755
--- a/drivers/net/phy/maxio.c
+++ b/drivers/net/phy/maxio.c
@@ -55,12 +55,12 @@ static int maxio_mae0621a_clk_init(struct phy_device *phydev)
 
 	if (0 == (workmode & BIT(5))) {
 		if (0 == (clkmode & BIT(8))) {
-			//oscillator
+			/* oscillator */
 			ret = phy_write_paged(phydev, 0xd92,
 					      MAXIO_MAE0621A_CLK_MODE_REG,
 					      clkmode | BIT(8));
 		} else {
-			//crystal
+			/* crystal */
 			ret = phy_write_paged(phydev, 0xd92,
 					      MAXIO_MAE0621A_CLK_MODE_REG,
 					      clkmode & ~BIT(8));
@@ -85,7 +85,7 @@ static int maxio_mae0621a_config_init(struct phy_device *phydev)
 	if (ret < 0)
 		return ret;
 
-	//adjust TX/RX delay
+	/* adjust TX/RX delay */
 	switch (phydev->interface) {
 	case PHY_INTERFACE_MODE_RGMII:
 		val = 0x0;
-- 
2.39.5

