From d2bcc2d2ba34417b2cf72a8fc9c01add34a54d10 Mon Sep 17 00:00:00 2001
From: Andrew Lunn <andrew@lunn.ch>
Date: Mon, 5 May 2025 18:47:29 -0500
Subject: [PATCH 23/30] net: phy: maxio: Remove unused headers, sort the
 remaining

Signed-off-by: Andrew Lunn <andrew@lunn.ch>
---
 drivers/net/phy/maxio.c | 7 ++-----
 1 file changed, 2 insertions(+), 5 deletions(-)

diff --git a/drivers/net/phy/maxio.c b/drivers/net/phy/maxio.c
index 6d2885a5f..ffd86c873 100755
--- a/drivers/net/phy/maxio.c
+++ b/drivers/net/phy/maxio.c
@@ -9,12 +9,9 @@
  * Copyright (c) 2004 maxio technology, Inc.
  */
 #include <linux/bitops.h>
-#include <linux/phy.h>
-#include <linux/module.h>
 #include <linux/delay.h>
-#include <linux/device.h>
-#include <linux/timer.h>
-#include <linux/netdevice.h>
+#include <linux/module.h>
+#include <linux/phy.h>
 
 #define MAXIO_PAGE_SELECT		0x1f
 #define MAXIO_MAE0621A_INER		0x12
-- 
2.39.5

