From 531c4b10675bc2afd16a809f4eaa57983af971ff Mon Sep 17 00:00:00 2001
From: Andrew Lunn <andrew@lunn.ch>
Date: Mon, 5 May 2025 19:09:16 -0500
Subject: [PATCH 27/30] net: phy: maxio: checkpatch: Fixup error messages

Signed-off-by: Andrew Lunn <andrew@lunn.ch>
---
 drivers/net/phy/maxio.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/net/phy/maxio.c b/drivers/net/phy/maxio.c
index bf5a73730..de1193787 100755
--- a/drivers/net/phy/maxio.c
+++ b/drivers/net/phy/maxio.c
@@ -99,7 +99,7 @@ static int maxio_mae0621a_config_init(struct phy_device *phydev)
 
 	ret = phy_read_paged(phydev, 0xd96, 0x0);
 	if (ret < 0) {
-		phydev_err(phydev, "Failed to update the TX delay register\n");
+		phydev_err(phydev, "Failed to read RGMII delay register\n");
 		return ret;
 	}
 
@@ -107,7 +107,7 @@ static int maxio_mae0621a_config_init(struct phy_device *phydev)
 
 	ret = phy_write_paged(phydev, 0xd96, 0x0, val | ret);
 	if (ret < 0) {
-		phydev_err(phydev, "Failed to update the TX delay register\n");
+		phydev_err(phydev, "Failed to write RGMII delay register\n");
 		return ret;
 	}
 
-- 
2.39.5

