From dad01e3d525f143612192715c670c3d6da92715e Mon Sep 17 00:00:00 2001
From: John Clark <inindev@gmail.com>
Date: Tue, 6 May 2025 20:07:00 -0400
Subject: [PATCH 30/30] net: phy: maxio: relax phy matching from exact to model

Signed-off-by: John Clark <inindev@gmail.com>
---
 drivers/net/phy/maxio.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/net/phy/maxio.c b/drivers/net/phy/maxio.c
index 56015d679..f9a8c10e5 100755
--- a/drivers/net/phy/maxio.c
+++ b/drivers/net/phy/maxio.c
@@ -141,7 +141,7 @@ static int mae0621a_probe(struct phy_device *phydev)
 
 static struct phy_driver mae0621_drvs[] = {
 	{
-		PHY_ID_MATCH_EXACT(0x7b744411),
+		PHY_ID_MATCH_MODEL(0x7b744411),
 		.name		= "MAE0621A Gigabit Ethernet",
 		.features	= PHY_GBIT_FEATURES,
 		.probe		= mae0621a_probe,
@@ -157,7 +157,7 @@ static struct phy_driver mae0621_drvs[] = {
 module_phy_driver(mae0621_drvs);
 
 static const struct mdio_device_id __maybe_unused mae0621a_tbl[] = {
-	{ PHY_ID_MATCH_EXACT(0x7b744411) },
+	{ PHY_ID_MATCH_MODEL(0x7b744411) },
 	{ }
 };
 
-- 
2.39.5

