From 896a02b367c6ccf8221b6b2c80c032ebff0d4c97 Mon Sep 17 00:00:00 2001
From: John Clark <inindev@gmail.com>
Date: Wed, 30 Apr 2025 09:01:09 -0400
Subject: [PATCH 5/7] ignore power domain links to soc

Signed-off-by: John Clark <inindev@gmail.com>
---
 drivers/base/core.c | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/drivers/base/core.c b/drivers/base/core.c
index cbc0099d8..9d2673edc 100644
--- a/drivers/base/core.c
+++ b/drivers/base/core.c
@@ -2115,6 +2115,13 @@ static int fw_devlink_create_devlink(struct device *con,
 	if (link->flags & FWLINK_FLAG_IGNORE)
 		return 0;
 
+	/* skip suppliers with the name 'soc' */
+	const char *sup_name = fwnode_get_name(sup_handle);
+	if (sup_name && sup_name[0] == 's' && sup_name[1] == 'o' && sup_name[2] == 'c' && sup_name[3] == '\0') {
+		dev_dbg(con, "Skipping link to supplier named 'soc' (%pfwf)\n", sup_handle);
+		return 0;
+	}
+
 	/*
 	 * In some cases, a device P might also be a supplier to its child node
 	 * C. However, this would defer the probe of C until the probe of P
-- 
2.39.5

